Makefile

```
@@ -14,13 +14,18 @@ check: ## Run code quality tools.
 	@echo "🚀 Checking for dependency issues: Running deptry"
 	@pdm run deptry python
 
-.PHONY: test
```

Can we keep this command to make it easier to run all unit tests locally?

```suggestion
.PHONY test:
test: test-unit test-functional

```